eslint.config.mjs

```diff
@@ -29,11 +29,6 @@ const restrictedImportPatterns = [
     group: ['sentry/components/devtoolbar/*'],
     message: 'Do not depend on toolbar internals',
   },
-  {
-    group: ['*.spec*'],
-    message:
-      'Do not import from test files. This causes tests to be executed multiple times.',
-  },
```

Comment:
This is replaced by https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-export.md